TestTools/TestData/TokenGenerator.swift

```diff
@@ -29,7 +29,28 @@ class TokenGenerator {
     
     static let shared = TokenGenerator()
     
-    func fetchToken(for userId: String, expiration tokenDurationInMinutes: Double = 0) -> UserToken? {
+    var retries = 0
+    
+    func fetchToken(for userId: String, expiration tokenDurationInSeconds: Int = 0) -> UserToken? {
+        // When tokens expires after #{expiration} seconds, what to do?
```

Comment:
Is this forgotten? If a token expires, the backend returns 401, while the SDK calls the token provider to refresh the token.

Reply:
Oops, these are development/debugging comments 😄
Deleted 👍